includes/formatting.php

```diff
@@ -57,6 +57,60 @@ function get_body_contents( $content ) {
 add_filter( 'airstory_before_insert_content', __NAMESPACE__ . '\get_body_contents', 1 );
 
 /**
+ * Sideload a single image from a remote URL.
+ *
+ * @param string $url      The remote URL for the image.
+ * @param int    $post_id  Optional. The post the newly-uploaded image should be attached to.
+ *                         Default is 0 (unattached).
+ * @param array  $metadata Optional. Additional post meta keys to assign once the attachment post
+ *                         has been created. These keys and values are assumed to be sanitized.
+ *                         Default is an empty array.
+ */
+function sideload_image( $url, $post_id = 0, $metadata = array() ) {
+	require_once ABSPATH . 'wp-admin/includes/media.php';
+	require_once ABSPATH . 'wp-admin/includes/file.php';
+	require_once ABSPATH . 'wp-admin/includes/image.php';
```

Comment:
I had the same thought, but [`download_url()` is defined in wp-admin/includes/file.php](https://developer.wordpress.org/reference/functions/download_url/#source) — in the interest of not having multiple `require_once` blocks, I've kept them together at the top.